pkg/api/job_runs.go

```diff
@@ -106,6 +111,7 @@ func FetchJobRun(dbc *db.DB, jobRunID int64, logger *log.Entry) (*models.ProwJob
 	// Load the ProwJobRun, ProwJob, and failed tests:
 	// TODO: we may want to expand to analyzing flakes here in the future
 	res := dbc.DB.Joins("ProwJob").
+		Preload("PullRequests").
```

Comment:
Was this something that is being used already and is performance boost or left over from previous work (I didn't see a new reference to pull requests but might be missing it.

Reply:
I believe Stephen added it here: https://github.com/openshift/sippy/commit/83eba80ed5518368e416fdbeb459dbb8ddce1c2a. Don't think it was used though.

Reply:
I just meant the Preload part, I missed why that was added but I think its `jobRun.PullRequests ` https://github.com/openshift/sippy/blob/383af8a28ec5ca37d526c8b73135f9577363c2e7/pkg/api/job_runs.go#L560